docs/source/extension/documents.rst

```diff
@@ -39,9 +39,12 @@ communicate with each other.
 Models contain an instance of `ModelDB <../api/classes/observables.modeldb-1.html>`__
 that acts as data storage for the model's content. In JupyterLab 3.1, we introduced
 the package ``@jupyterlab/shared-models`` to swap ``ModelDB`` as a data storage
-to make 'documents' collaborative. We implemented these shared models using
-`Yjs <https://docs.yjs.dev>`_, a high-performance CRDT for building collaborative applications
-that automatically sync.
+to make 'documents' collaborative, and in JupyterLab 3.6 we moved this package to
+the `jupyter_ydoc <https://github.com/jupyter-server/jupyter_ydoc>`__ repository
+and renamed it as ``@jupyter/ydoc`` to keep in sync the front-end and back-end
+implementations of the shared models. We implemented these shared models using
+`Yjs <https://docs.yjs.dev>`_, a high-performance CRDT for building collaborative
+applications that automatically sync.
 At the moment, models contain both a ``ModelDB`` and a ``Shared Model`` instance, so it is
 possible to access ``ModelDB`` yet.
```

Comment:
As this is targeting master first, let's focus on that version and change the explanation when backporting.

```
Models contain an instance of `ISharedDocument <..link..>`_ that acts as data storage for the model's
content. As of JupyterLab 4, the default data storage implementation is a `YDocument <..link..>`_ based on
`Yjs <https://docs.yjs.dv>`_, a high-performance CRDT for building collaborative applications. Both the
interface and the implementation are provided by the package `@jupyter/ydoc <https://github.com/jupyter-server/jupyter_ydoc>`_.
```